jdisc_core/pom.xml

```diff
@@ -104,6 +104,7 @@
             </exclusions>
         </dependency>
         <dependency>
+            <!-- TODO: Remove, we don't use commons-daemon anymore, and should not provide it.  -->
```

Comment:
TODO Vespa 8?

Reply:
No, I don't think anybody is using it. It's not imported by any hosted bundles.